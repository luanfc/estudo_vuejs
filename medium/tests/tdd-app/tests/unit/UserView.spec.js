import { shallowMount } from '@vue/test-utils'
import UserView from '@/views/UserView'

describe('UserView', () => {
    it('renders the components', () => {
        const wrapper = shallowMount(UserView)
        expect(wrapper.html()).toMatchSnapshot()
    })
})