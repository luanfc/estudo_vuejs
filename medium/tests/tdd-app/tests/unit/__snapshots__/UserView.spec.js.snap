// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserView renders the components 1`] = `
<div>
  <vusersearchform-stub></vusersearchform-stub>
  <vuserprofile-stub user="[object Object]"></vuserprofile-stub>
</div>
`;
